ui/client/components/graph/PanZoomPlugin.ts

```diff
@@ -1,9 +1,10 @@
 import {dia} from "jointjs"
+import {debounce} from "lodash"
 import svgPanZoom from "svg-pan-zoom"
 import {CursorMask} from "./CursorMask"
 import {Events} from "./joint-events"
 
-type EventData = { panStart?: { x: number, y: number } }
+type EventData = { panStart?: { x: number, y: number, touched?: boolean } }
```

Comment:
What does mean touched? Maybe better will be `chaged`?

Reply:
touched - dirty - used - I need this state to filter out random movements